tests/unit/utils/MockDataForSVM.h

```diff
@@ -0,0 +1,83 @@
+#include <shogun/labels/BinaryLabels.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/features/DataGenerator.h>
+
+using namespace shogun;
+
+class CMockDataForSVM
```

Comment:
I think this should be added in a sep. PR.
Also, why is this just for SVM? This is generally linearly separable data with binary labels....